README.md

```
@@ -1,129 +1,219 @@
+[![Swift](https://img.shields.io/badge/Swift-F54A2A?style=for-the-badge&logo=swift&logoColor=white)](https://swift.org/)
+[![Shell](https://img.shields.io/badge/Shell_Script-121011?style=for-the-badge&logo=gnu-bash&logoColor=white)](https://www.gnu.org/software/bash/)
+<br/>Provider • Bloc • Getx • Riverpoad • Domain-Driven Design (DDD)
+
+</div>
+
+---
+
+## 💡 My Development Philosophy
+
+<div align="center">
+<img src="https://github.com/sindresorhus/sindresorhus/blob/main/cat-typing.gif" width="200" />
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)